src/components/LicenseRemindModal/index.jsx

```diff
@@ -181,9 +182,11 @@ class LicenseRemindModal extends React.Component {
     return (
       <React.Fragment>
         <span className="d-block">{title}</span>
-        <small>
-          {subtitle}
-        </small>
+        {subtitle && (
```

Comment:
`LicenseRemindModal` component required `subtitle` but the two places we were using `LicenseRemindModal` we were only passing an empty string. So, conditionally render this and make `subtitle` optional on the component.